app/javascript/react/components/navbar.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router'
+import BackButton from './BackButton'
+
+const NavBar = (props) => {
+
+  return(
+    <div>
+      <BackButton />
+      <Link to='/'>Home</Link>
+      <h1 className="page-title">Our Wizards:</h1>
```

Comment:
Do you want `Our Wizards` to show up on every page, or only on the Wizards Index page? If it's in the NavBar, it will be on each page.